src/otx/core/model/entity/classification.py

```diff
@@ -305,6 +308,14 @@ def _create_model(self) -> nn.Module:
         self.classification_layers = classification_layers
         return model
 
+    def set_hlabel_info(self, hierarchical_info: HLabelInfo) -> None:
+        """Set hierarchical information in model head.
+
+        Args:
+            hierarchical_info: the label information represents the hierarchy.
+        """
+        self.model.head.set_hlabel_info(hierarchical_info)
```

Comment:
```suggestion
```

Reply:
If you remove it -> hierarchical classification won't work

Reply:
Why do you remove it? without it -> hierarchical classification won't work

Reply:
Sorry my mistake. I forgot  to remove this comment. I wanted https://github.com/openvinotoolkit/training_extensions/pull/2815#discussion_r1465744303 for `set_hlabel_info`.